app/src/fagord-main.tsx

```diff
@@ -1,46 +1,52 @@
-import 'bootstrap/dist/css/bootstrap.min.css';
-import './index.css';
-
-import { createTheme, ThemeProvider } from '@mui/material/styles';
+import { Navigate, Route, Routes } from '@remix-run/react';
+import { ContactPage } from '~/src/pages/contact-page/contact-page';
+import { AboutPage } from '~/src/pages/about-page/about-page';
+import { DictionaryPage } from '~/src/pages/dictionary-page/dictionary-page';
+import { ArticlePage } from '~/src/pages/article-page/article-page';
+import { Article } from '~/src/pages/article-page/article/article';
 import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
-import { ReactQueryDevtools } from '@tanstack/react-query-devtools';
-import { StrictMode } from 'react';
-import ReactDOM from 'react-dom/client';
-import { BrowserRouter } from 'react-router-dom';
-
-import { Router } from './pages/router';
-
-const queryClient = new QueryClient({
-  defaultOptions: {
-    queries: {
-      retry: false,
-    },
-  },
-});
+import { featureToggles } from '~/src/utils/feature-toggles.client';
+import { Header } from '~/src/components/header/header';
+import { ClientOnly } from 'remix-utils/client-only';
+import { useState } from 'react';
+import { HomePage } from '~/src/pages/home-page/home-page';
+import { PageAlert } from '~/src/components/page-alert/page-alert';
+import { TermPage } from '~/src/pages/term-page/term-page';
+import { NewTermPage } from '~/src/pages/new-term-page/new-term-page';
 
-const fagordTheme = createTheme({
-  palette: {
-    primary: {
-      main: '#2e9cca',
-    },
-    secondary: {
-      main: '#9eb9ca',
-    },
-    background: {
-      paper: '#fbfcfd',
+export default function App() {
+  const [queryClient] = useState(() => new QueryClient({
```

Comment:
Denne er nødvendig for å sikre at én query-tilstand ikke deles av flere klienter: https://www.reddit.com/r/reactjs/comments/1aracyz/should_i_initialize_queryclient_of_reactquery/